backend/src/server.ts

```diff
@@ -0,0 +1,7 @@
+import app from './express';
+
+const port = process.env.PORT || 3000;
```

Comment:
The Quickstart template may contradict me, so take any specifics with a grain of salt if the template says otherwise. A couple of things based on this one. 

- Use a more descriptive name like `API_PORT`, your root folder will might become a large ENV list and you'll have to differentiate things like Database Ports, API Ports and the like. 

- You'll want/need something like `dotenv` or equivalent installed and configured, currently if I add an `ENV` variable it's still going to use the default argument you provided me.

- I'm going to recommend right now that you start adding a `.env.template` file in the root folder and subdirs of this repo to keep track of what env variables are being used and where. It's a good way to communicate to outsiders and in your team when/where new variables are being added and used, and helps reduce a lot of troubleshooting in figuring it out. It would look a lot like an `.env` file, but you only supply the keys, not the variables
    - You can also add a README to the API Folder and display your keys in a table to write descriptions (as an option)
    - Many repos will use make commands and bash scripts to create templated `.env`s There's plenty of options! You just have to remember to update them as you progress

README Example:

| Key          | Example    | Purpose                                                              |
| ---           | ----           | ----                                                                    |
| API_HOST|  'localhost' | Used to determine current hosting environment |

`.env.template` example:

```
API_HOST=
API_PORT=
```

Nice use of the backup argument too :)